blog/_posts/2018-07-09-introducing-TeamBarcelonaDuo.md

```diff
@@ -0,0 +1,68 @@
+
+### Amalia
+I was born in Mexico, but I moved to California when I was 5 years old. I met my partner while living in San Francisco and moved to Barcelona in 2008. I am based in Barcelona but spend part of the year in California and in Greece.
+
+Prior to programming, my background was in research in the media studies/ sociology field. I conducted research on [ alternative economic practices in Catalunya](https://global.oup.com/academic/product/aftermath-9780199658411?cc=ru&lang=en&), the Indignado and Occupy Wallstreet movement, [the banking crisis in Spain and Iceland](http://in3-working-paper-series.uoc.edu/in3/en/index.php/in3-working-paper-series/article/view/1943.html), and I also researched how [trust is built online](http://www.lse.ac.uk/media-and-communications/assets/documents/events/phd-symposium/Brochure-with-abstracts-PhD-Symposium-2014.pdf)
+
+
+**Why I decided to learn to program**
+
+While in graduate school, through my research I started to see how the criteria that decide if somebody is seen as trustworthy online are decided by a few programmers. I became concerned to realize that in the era of big data and the increasing use of algorithms to sort, calculate and make decisions for us, the concerns of people that are already marginalized offline are not taken into account because of the lack of diversity among those creating the technology. I decided that in order to help change this imbalance I myself should learn to program.
+
+Even though I am not the next Mark Zuckerberg, I want to increase diversity in the tech space and this is why I want to become a better programmer and be part of the programming community. In my spare time, I help organize the codebar Barcelona chapter. Codebar is a non-profit initiative that runs regular programming workshops for underrepresented people in tech.
+
+![Amalia coaching at codebar!](/img/blog/2018/BarcelonaDuo6.jpg)
+<div class="image-credits"><b>Amalia coaching at codebar (image by Amalia Cardenas)</b></div>
+<br>
+
+This summer is very important for me. I know that programming takes a lot of hard work to learn, but it is something that now I can see myself doing. I know that as I get better, I will inspire others that they can get better too. I look forward to seeing where this summer will take me. :)  
+
+### What we will be working on this summer
+
+Exercism.io is a platform that helps people achieve fluency in a programming language through individual practice and mentor feedback. Students can choose over 45 different languages to master.
+
+Lori and I will be focusing on mentors, mentor resources, and tooling for mentors. We hope to develop resources that can make the process of giving feedback smoother for mentors and increase the quality of feedback across all of the exercises available on the platform.
+
+
+ **Follow us on Twitter:** [@HarlemExpat](https://twitter.com/HarlemExpat), [@anybodycancode1](https://twitter.com/anybodycancode1)
```

Comment:
Probably it's good to write who is who on twitter. May be instead of using twitter ids as a link text, use your names.